src/ralph/lib/transitions/apps.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from django.apps import AppConfig
+
+from ralph.lib.transitions.models import update_models_attrs
+
+
+class TransitionAppConfig(AppConfig):
+    name = 'ralph.lib.transitions'
+    verbose_name = 'Transitions'
+
+    def ready(self):
+        try:
+            # RuntimeError raised when database is empty (e.g. after tests)
```

Comment:
please elaborate more about this workaround (and possible solutions in the future as we discussed).